server/serialization.go

```diff
@@ -0,0 +1,22 @@
+package server
+
+import (
+	"encoding/base64"
+	"encoding/json"
+
+	"github.com/coniks-sys/coniks-go/merkletree"
+)
+
+var b64en = base64.StdEncoding.EncodeToString
+
+func MarshalTemporaryBinding(tb *merkletree.TemporaryBinding) ([]byte, error) {
+	return json.Marshal(&struct {
```

Comment:
The TB needs to include the STR proof and the index proof if this is what is being returned to the client.